autonomx/ComputeEngine.cpp

```diff
@@ -182,16 +182,31 @@ void ComputeEngine::loop() {
 
     // send output messages to osc engine
     for(QList<QSharedPointer<Generator>>::iterator it = generatorsList->begin(); it != generatorsList->end(); it++) {
-        QList<QVariant> outputs;
+        QList<QVariant> oscOutputsList;
+        QList<QVariant> oscOutputRegion;
+
         for(int i = 0; i < (*it)->getOutputRegionSet()->rowCount(); i++) {
             double value = flagDummyOutputMonitor ? randomUniform(randomGenerator) : (*it)->getOutputRegionSet()->at(i)->getIntensity();
             if(flagCastOutputToFloat) {
-                outputs.append((float)value);
+                oscOutputsList.append((float)value);
+                oscOutputRegion.append((float)value);
             } else {
-                outputs.append(value);
+                oscOutputsList.append(value);
+                oscOutputRegion.append(value);
             }
+
+            // write and send individual output region osc messages
+            QString outputRegionNumber = QString::number(i+1);
+            oscOutputRegion.prepend("/" + (*it)->getGeneratorName() + "/output/" + outputRegionNumber);
+            emit sendOscData((*it)->getID(), oscOutputRegion);
```

Comment:
Can you add a comment saying what is the format of the OSC message(s) sent here?